docs/writing-policies/spec/host-capabilities/06-kubernetes.md

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_label: "Kubernetes Capabilities"
+title: ""
+---
+
+# Kubernetes Capabilities
+
+Kubewarden context aware policies requires access to resources from the Kubernetes
+cluster where they are running on. For that, the Kubewarden SDKs expose functions
+that uses the WaPC communication protocol to talk with the host system asking for
+data about the cluster.
+
+## WaPC protocol contract
+
+Under the hood, the SDKs functions uses  two functions available in the runtime
+that handle this kind of  information request. They are:
+
+- `list_resources_by_namespace` : WaPC function to request resources information from a given namespace
+- `list_resources_all`: WaPC function to request resources information from the whole cluster
+
+This is the description of the input and output payloads used by the WaPC protocol used to expose this capability:
+
+<table>
+<tr>
+<th> WaPC function name </th> <th> Input payload </th> <th> Output payload </th>
+</tr>
+<tr>
+<td>
+
+`list_resources_all`
+
+</td>
+<td>
+
+```hcl
+{
+	# API Group version. Use `v1` for the `core` group and `groupName/groupVersion` for all other groups
+	"api_version": string,
+	# Resource kind
+	"kind": string,
+	# Label selector to filter the resources
+	"label_selector": string,
+	# Field selector to filter the resources
+	"field_selector": string
+}
+```
+
+</td>
+<td>
+
+Result of <code>GET /apis/$api_version/$kind</code>
```

Comment:
```suggestion
Return a Kubernetes [`List`](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#types-kinds), which is a collection of Kubernetes objects of the same type.
```

Reply:
I would also add a `info` block (I hope it works here given all the funky stuff we're doing with the html table) that states that this API has to be used to get cluster-wide resources like Namespace